### ISSUE TITLE
gogoanime video/video player not loading

### ISSUE THREAD

**Issue by re11ding** [Karmesinrot/Anifiltrs#36]
This could be a temporary issue with the site, but since gogoanime switched domains to .so, the Anifiltrs list is preventing videos from loading, regardless of server. This has been tested with both all other addons enabled and disabled and the result remains the same. Disabling the main Anifiltr fixes the issue and allows videos to load. It appears to be blocking the player itself, not so much the servers it gets the videos from.

**Comment by Karmesinrot** [comment]
Gogoanime now uses gogo-stream.com for the videos I've pushed a commit to fix it. With the exception of the cloud9 server, which seems to be down atm, the video servers will load.

**Comment by re11ding** [comment]
Thanks again! I appreciate your dedication to keeping this list working. I'd suggest what to do to fix it myself had I any understanding how to figure these conflicts out to begin with.

**Comment by Karmesinrot** [comment]
The breakage was due to gogo-stream.com not being on the [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) filter so I just had to include it and other related resources. CSP filters have the pros and cons, one of it's cons are being difficult to maintain and possible brekage when a site uses a new resource such as in this case. The filter is one of the results of my continuous experimentation with filter writing.
